q2_metadata/plugin_setup.py

```diff
@@ -22,3 +24,21 @@
                  'and visualizing Metadata.'),
     short_description='Plugin for working with Metadata.'
 )
+
+plugin.visualizers.register_function(
+    function=tabulate,
+    inputs={},
+    parameters={
+        'input': qiime2.plugin.Metadata,
+        'page_size': qiime2.plugin.Int,
+    },
+    parameter_descriptions={
+        'input': 'The metadata to tabulate.',
+        'page_size': 'The maximum number of Metadata records to display '
+                     'per page',
+    },
+    name='tabulate',
```

Comment:
Looking at existing `names`, there doesn't seem to be a language/style convention... so whatever you think sounds and looks good :+1: